script/update-aws-cdk

```diff
@@ -34,6 +34,10 @@ updateAwsCdk() {
   npm run update-aws-cdk
 }
 
+installAwsCdkUpdates() {
+  npm install && cd integration-test && npm install
```

Comment:
> It also looks like this is a v2 lockfile, which comes from npmv7. Is this desired? 

When this part was being run from within `package.json`, npmv7 was being used. However:
```
npm --version
6.14.8
```
😕

Reply:
🤔  that _is_ odd!

Reply:
Do we need to `cd ..` after this too to stay at the root? Alternatives:

```sh
(
  cd integration-test
  npm install
)
```

Or `pushd`/`popd`.